spec/lib/tasks/publishing_api_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "rails_helper"
+require "rake"
+
+RSpec.describe "publishing_api.rake" do
+  it "should publish special routes" do
+    publishing_api_url = "http://publishing-api.dev.gov.uk".freeze
+    content_id = "bce40c1f-2259-4404-b275-8c5e04afef34".freeze
+
+    Rake.application = Rake::Application.new
+    Rake.application.rake_require("lib/tasks/publishing_api", [Rails.root.to_s], [])
+    Rake::Task.define_task(:environment)
```

Comment:
You can avoid needing to do this (and requiring rake) if you use `Rails.application.load_tasks`, typically that ends up in the rails_helper file as it's only safe to call once.

Reply:
Nice! Today I learned.